src/ui/profile_preferences.ui

```diff
@@ -428,18 +425,18 @@
             </property>
            </widget>
           </item>
-          <item row="2" column="0" colspan="3">
-           <widget class="QCheckBox" name="auto_clear_input_line_checkbox">
-            <property name="enabled">
-             <bool>true</bool>
+          <item row="1" column="2" colspan="2">
+           <widget class="QCheckBox" name="checkBox_runAllKeyBindings">
+            <property name="toolTip">
+             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Check all Key-bindings against key-presses.&lt;/p&gt;&lt;p&gt;&lt;i&gt;Versions of Mudlet prior to &lt;b&gt;3.9.0&lt;/b&gt; would stop checking after the first matching combination of&lt;/i&gt; KeyCode &lt;i&gt;and&lt;/i&gt; KeyModifier &lt;i&gt;was found and then send the command and/or run the script of that Key-binding only.  This&lt;/i&gt; per-profile &lt;i&gt;option tells Mudlet to check and run the remaining matches; but, to retain compatibility with previous versions, defaults to the &lt;b&gt;un&lt;/b&gt;-checked state.&lt;/i&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
             <property name="text">
-             <string>Auto clear the input line after you sent text</string>
+             <string>Use all Key-bindings on same key</string>
```

Comment:
Well, we are talking about a particular type of Mudlet ***Item*** - on the main toolbar button we just say **Keys** but in other places we say *Key-bindings* - I would tend to use Title-Case because we are talking about a feature of Mudlet like: *Aliases*, *Timers* or *Scripts*...

"Responds to" *could* work - and we do have to be careful with using short Americanish words because other languages may not have an equally short equivalent - but it would not have been my first choice - as you may be able to tell!

Anyone else have an opinion?

Reply:
Yeah, it sounds nice:

> Respond to all keybindings on the same key

Reply:
Looking at my other comment, there I suggested `React to` instead. Maybe that sounds even nicer, SlySven?

Reply:
Done...